CMakeLists.txt

```diff
@@ -297,7 +297,7 @@ add_subdirectory(library)
 add_subdirectory(pkgconfig)
 
```

Comment:
Missing: apply `code_style.py` (and more?) to framework files (https://github.com/Mbed-TLS/mbedtls-framework/issues/7).